packages/react-reconciler/src/SchedulingProfiler.js

```diff
@@ -59,32 +71,52 @@ if (enableSchedulingProfiler) {
   }
 }
 
-export function formatLanes(laneOrLanes: Lane | Lanes): string {
-  let labels = getLabelsForLanes(laneOrLanes);
-  if (labels != null) {
-    labels = labels.sort().join(',');
-  } else {
-    labels = '';
+const laneLabels: Array<string> = [];
+
+export function getLaneLabels(): Array<string> {
```

Comment:
so we can change all of these apis (e.g. getLabelForLane or this new getLaneLabels) within reconciler without worrying about breaking changes because they're all internal to reconciler, right?

the only thing that's actually braking is the output format, which we check for using the SUPPORTED_PROFILER_VERSION on the other side, right?

Reply:
That's correct 👍🏼